Code/autopkglib/recipes/__init__.py

```diff
@@ -0,0 +1,762 @@
+#!/usr/local/autopkg/python
+#
+# Copyright 2023 Nick McSpadden
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import json
+import os
+import pathlib
+import plistlib
+import pprint
+import sys
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional
+
+import yaml
+
+sys.path.append("/Users/nmcspadden/Documents/GitHub/autopkg/Code")
```

Comment:
Whatever this was a quick-and-dirty workaround for; you should remove it now.

Reply:
Removed! Thankfully nothing broke...